snippets/as/simple-storage/src/__tests__/e2e/types.ts

```diff
@@ -0,0 +1,24 @@
+// $start: js-e2e-test-types
+export type UInt8 = number;
```

Comment:
We should probably use the `w3 app codegen` command to generate these types. Checkout an example here: https://github.com/polywrap/monorepo/tree/prealpha/packages/templates/api/assemblyscript/src/__tests__/types

Reply:
done